README.md

```diff
@@ -164,3 +179,28 @@ by Fika.
 If you'd like to be part of the Fika community, please consider joining
 our [Discord server](https://discord.gg/zNs6Gs5). That's the best place to
 ask questions or get guidance on contributing to Fika.
+
+## License
+
+Licensed under the [MIT License](LICENSE)
```

Comment:
Please change this to **Apache License 2.0**

_You could word it something like:_


Fika source code is released under Apache License 2.0.
Check [LICENSE](https://github.com/fika-lang/fika/blob/master/LICENSE) for more information.

Reply:
Ok